ibm/service/power/ibm_pi_constants.go

```diff
@@ -0,0 +1,52 @@
+package power
+
+const (
+	PICloudInstanceID = "pi_cloud_instance_id"
+
+	// ---------- Image ----------
+	// Arguments
+	PIImageName                  = "pi_image_name"
+	PIImageAffinityInstance      = "pi_affinity_instance"
+	PIImageAffinityPolicy        = "pi_affinity_policy"
+	PIImageAffinityVolume        = "pi_affinity_volume"
+	PIImageAntiAffinityInstances = "pi_anti_affinity_instances"
+	PIImageAntiAffinityVolumes   = "pi_anti_affinity_volumes"
+	PIImageID                    = "pi_image_id"
+	PIImageBucketName            = "pi_image_bucket_name"
+	PIImageAccessKey             = "pi_image_access_key"
+	PIImageBucketAccess          = "pi_image_bucket_access"
+	PIImageBucketFile            = "pi_image_bucket_file_name"
+	PIImageBucketRegion          = "pi_image_bucket_region"
+	PIImageSecretKey             = "pi_image_secret_key"
+	PIImageStoragePool           = "pi_image_storage_pool"
+	PIImageStorageType           = "pi_image_storage_type"
+	CatalogImagesSAP             = "sap"
+	CatalogImagesVTL             = "vtl"
+
+	// Attributes
+	Images               = "image_info"
+	CatalogImages        = "images"
+	ImagesID             = "id"
```

Comment:
For common strings like id and name do you want just use the same constants rather than have the same strings defined over and over for the different resource types?

Reply:
For now I think it's better to redefine the strings. We can easily review and see all arguments/attributes for a resource type.